aten/src/ATen/native/README.md

```diff
@@ -299,6 +299,18 @@ with other components of PyTorch in order to reduce overall complexity.
 If you find yourself having to set this field to False add @gchanan to your PR's
 set of reviewers.
 
+### `use_c10_dispatcher`
+
+```
+use_c10_dispatcher: True
+```
+
+This will indicate that the func signature only uses features supported by
```

Comment:
It's not that important that people modifying native_functions.yaml undersatnd your plan for the c10 dispatcher, but it is important that they understand what to do for their change.  So you should add some guidance like:
1) mention the current limitations
2) suggest if the func doesn't have those limitations, to set the flag
3) mention if everything compiles, that it should be fine to use dispatcher, otherwise turn it off

Something like that.

Reply:
updated it

Reply:
I don't see any guidance about current limitations in the doc here?